modules/mobile/app/models/mobile/v0/adapters/vaos_v2_appointment.rb

```diff
@@ -162,8 +162,10 @@ def extract_station_and_ien(appointment)
           identifier[:value]&.split(':', 2)&.second
         end
 
```

Comment:
I'm not sure if this comment is helpful, but I spent quite a bit of time checking out how the web FE does this and concluded that without comparing our mocks and theirs for each appointment type, it's probably impossible to be sure this 100% correct and thought it was worth noting the uncertainty. 🤷